### ISSUE TITLE
Consider implement a messege queue to make publish api as an asynchronous call

### ISSUE THREAD

**Issue by feng-lu-veracity** [veracity/DataLineage-Viewer#16]
Today the publish api is a synchronous call, the client has to wait until the submit is done, before it can send another data to tangle. 

consider to introduce message queue, to make publish api as an asynchronous call.

it might require redis or similar as the messaging system. lets discuss it before we start implementation. 

This is low priority.
